### DISCUSSION TITLE
How to parse dimensions in DXF file?

### DISCUSSION THREAD

**Discussion by chenneng** [mozman/ezdxf#441] [Q&A]
how to parsing dimension in DXF file? is there any example?

**Comment by mozman** [comment]
You have to be more specific. The DIMENSION entity is a complex entity with 8 different types and each type changes the meaning of the definition points. The associated DIMSTYLE entity has more than hundred DXF attributes and each DIMENSION entity can override each attribute individually. The graphical content is stored in an anonymous block in the BLOCKS section.

Frankly, this is not a task for a beginner!
